src/items/SizingWrapper.tsx

```diff
@@ -30,7 +30,7 @@ export const SizingWrapper = ({
 }): JSX.Element => {
   const width = getWidthFromSizing(size);
   return (
-    <Box maxWidth='100%' width={width}>
+    <Box maxWidth='100%' width={width} flex={1}>
```

Comment:
I would be careful here, why do you need flex=1? Add a comment, and could you please try out many different item types to be sure sizing is not broken?